pages/listings/listing/index.html

```diff
@@ -1,14 +1,56 @@
+    <meta name="description" content="" />
+
+    <link rel="stylesheet" href="/dist/css/index.css" />
+
+    <!-- Link to my font awesome, will probably have to be replaced with one that belongs to the entire team? -->
+    <script src="https://kit.fontawesome.com/155738729b.js" crossorigin="anonymous"></script>
+
+    <!-- Page title of single listing will have to be rendered in JS to get correct info related to the listing -->
     <title>PAge title</title>
-</head>
-<body id="homePage">
+  </head>
+  <body id="homePage">
     <header></header>
-    <main></main>
+    <main class="min">
+      <div class="container">
+        <div class="row d-flex alig-items-center justify-content-center my-5">
+          <div class="col-lg-7 d-md-flex align-items-start">
+            <!-- Back button, need to add JS event to make it take you to previous page -->
+            <button class="btn border-0 p-1 me-3 my-2 my-md-0 fs-5"><i class="fa-solid fa-chevron-left"></i></button>
+
+            <div class="card bg-theme-light border-0">
+              <!-- Content will be moved to be rendered in JS, just hard coded for now -->
+              <img class="card-img-top" src="" alt="Placeholder" />
+              <div class="card-body d-flex flex-column gap-1 p-sm-3 p-md-5">
+                <h1 class="card-title">Headline title</h1>
+                <small>Job title <span>&#x2022;</span> Company name <span>&#x2022;</span> Location</small>
+                <small>1 day ago <span>&#x2022;</span> Deadline date <span>&#x2022;</span> 47 Applicants</small>
+                <div class="d-flex gap-3 my-3">
+                  <button class="btn btn-theme-primary text-uppercase text-white w-100">Apply Now</button>
+                  <button class="btn btn-theme-secondary"><i class="fa-regular fa-heart"></i></button>
+                </div>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Error, vel consequuntur. Perferendis minus quidem quia consequatur accusamus laudantium quos atque fuga quae, libero nemo aliquid excepturi in unde dolorem enim!</p>
+                <ul>
+                  <li>Lorem</li>
+                  <li>Lorem</li>
+                  <li>Lorem</li>
+                </ul>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Earum atque perspiciatis, explicabo inventore quo voluptatibus officiis numquam qui nulla illum impedit fuga perferendis et necessitatibus hic esse nostrum sit est.</p>
+                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Praesentium ut velit recusandae, perferendis quas quo quia impedit vitae nobis quod, illum quisquam tempora ipsa consequuntur est tenetur esse cumque sunt?</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
     <footer></footer>
-</body>
-</html>
\ No newline at end of file
+    <script src="/node_modules/bootstrap/dist/js/bootstrap.bundle.min.js"></script>
```

Comment:
missing link to index.js in root folder.

Reply:
I think this should be pushed now